### PR TITLE
Add the homepage dynamically from API and add styles

### PR DESCRIPTION
**Pull request by samwel-mongare**
### The branch home-page_setup has the following features to be merged into the development branch:

- When the page loads, data is displayed from the Yu-gi-oh API.
- Styles in accordance with the wireframe provided.

### CODE REVIEW

**Comment by samwel-mongare** [comment]
Pull Request closes #15

File: src/modules/module1.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+const home = document.getElementById('home_link');
+
+const LOCAL_STORAGE_LIST_KEY = 'card.list';
+const cards = JSON.parse(localStorage.getItem(LOCAL_STORAGE_LIST_KEY)) || [];
+
+const clearElement = ((element) => {
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+});
```
Comment (o-rashad) [inline src/modules/module1.js]:
```suggestion
element.innerHTML = '';
```
- [ ] [OPTIONAL] This also would clear the element content in one line :+1:

File: src/modules/module1.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  }
+  const result = getCards();
+  return result;
+});
+
+const displayElements = (() => {
+  const blueEyes = document.getElementById('card_list');
+  clearElement(blueEyes);
+  for (let i = 0; i < cards.length; i++) {
+    const blueEye = document.getElementById('card_list');
```
Comment (o-rashad) [inline src/modules/module1.js]: The declared `blueEyes` variable on line 31 can be used here, its the same element :ok_hand:

**Review by o-rashad — APPROVED** [review]
The code look nice to me, 
keep up the good work

**Pull request merged**